sumo_rl/environment/env.py

```diff
@@ -207,7 +246,88 @@ def _discretize_elapsed_time(self, elapsed):
         for i in range(self.max_green//self.delta_time):
             if elapsed <= self.delta_time + i*self.delta_time:
                 return i
-        return self.max_green//self.delta_time -1
+        return self.max_green//self.delta_time - 1
 
 
-    
+class SumoEnvironmentPZ(AECEnv, EzPickle):
+    metadata = {'render.modes': [], 'name': "sumo_rl_v0"}
```

Comment:
'human' should be in the render modes list

Reply:
Unless the base environment truely has no rendering support I suppose.

Reply:
To use SUMO-GUI you have to initialize the env with use_gui=True.
Unfortunately, there is no way to make the rendering and the env logic independent with SUMO. 
So render() do nothing. Should I add 'human' to metadata anyway?